src/Tasks/Common/ConflictResolution/ConflictResolver.cs

```diff
@@ -66,30 +85,69 @@ public ConflictResolver(PackageRank packageRank, PackageOverrideResolver<TConfli
                         // replace existing item
                         if (commitWinner)
                         {
-                            winningItemsByKey[itemKey] = conflictItem;
+                            _winningItemsByKey[itemKey] = conflictItem;
                         }
                         else
                         {
-                            winningItemsByKey.Remove(itemKey);
+                            _winningItemsByKey.Remove(itemKey);
                         }
                         loser = existingItem;
-
                     }
 
-                    foundConflict(loser);
+                    foundConflict(winner, loser);
+
+                    //  If there were any other items that tied with the loser, report them as conflicts here
+                    List<TConflictItem> previouslyUnresolvedConflicts;
+                    if (_unresolvedConflictItems.TryGetValue(itemKey, out previouslyUnresolvedConflicts))
+                    {
+                        //  Skip the first item in the list of items that had unresolved conflicts, as it should
+                        //  be the same as the losing item which was just reported.
+                        foreach (var previouslyUnresolvedItem in previouslyUnresolvedConflicts.Skip(1))
```

Comment:
Can you use a ReferenceEquals here instead?  That would make me more comfortable than relying on list order.